backend/test/integration/healthz_api_test.go

```diff
@@ -0,0 +1,64 @@
+package integration
+
+import (
+	"testing"
+
+	"github.com/golang/glog"
+	"github.com/kubeflow/pipelines/backend/src/common/client/api_server"
+	"github.com/kubeflow/pipelines/backend/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type HealthzApiTest struct {
+	suite.Suite
+	namespace     string
+	healthzClient *api_server.HealthzClient
+}
+
+// Check the namespace have ML job installed and ready
+func (s *HealthzApiTest) SetupTest() {
+	if !*runIntegrationTests {
+		s.T().SkipNow()
+		return
+	}
+
+	if !*isDevMode {
+		err := test.WaitForReady(*namespace, *initializeTimeout)
+		if err != nil {
+			glog.Exitf("Failed to initialize test. Error: %v", err)
+		}
+	}
+	s.namespace = *namespace
+	clientConfig := test.GetClientConfig(*namespace)
+	var err error
+	s.healthzClient, err = api_server.NewHealthzClient(clientConfig, false)
+	if err != nil {
+		glog.Exitf("Failed to get healthz client. Error: %v", err)
+	}
+	s.cleanUp()
+}
+
+func (s *HealthzApiTest) TearDownSuite() {
+	if *runIntegrationTests {
+		if !*isDevMode {
+			s.cleanUp()
+		}
+	}
+}
+
+func (s *HealthzApiTest) cleanUp() {
+}
+
+func (s *HealthzApiTest) TestHealthzAPI() {
+	t := s.T()
+
+	/* ---------- Verify healthz response ---------- */
+	healthzResp, err := s.healthzClient.GetHealthz()
+	assert.Nil(t, err)
+	assert.NotNil(t, healthzResp)
```

Comment:
Can we check the content within healthzResp are json parsable?

Reply:
To show more about what I mean: For example: you can check the content within healthzResp contains `multi_user` key and value: https://github.com/kubeflow/pipelines/blob/74c7773ca40decfd0d4ed40dc93a6af591bbc190/backend/api/python_http_client/docs/ApiGetHealthzResponse.md.

Reply:
@zijianjoy thanks for the reply, but I think I still don't get it.
the question here is `healthzResp` is a well-defined structure, you can check the structure definition here 
https://github.com/kubeflow/pipelines/blob/74c7773ca40decfd0d4ed40dc93a6af591bbc190/backend/api/go_client/healthz.pb.go#L44
so the field `multi-user` will be there. 

But I think the question is: are we able to verify the field value `multi-user` to be true in a multi-tenancy setup. (as the multi-user field is a simple boolean field, we are unable to verify when the value is not presented.)
And still, it is hard to verify during this test (as we should be able to create a test environment w/o multiuser).